### PR TITLE
Feature/mat 5678 qdm testcase list calc

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE PR

Jira Ticket: [MAT-5678](https://jira.cms.gov/browse/MAT-5678)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/api/QdmCalculationService.ts [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
+export enum CqmPopulationType {
+  IPP = "initialPopulation",
+  DENOM = "denominator",
+  DENEX = "denominator-exclusion",
+  DENEXCEP = "denominator-exception",
+  NUMER = "numerator",
+  NUMEX = "numerator-exclusion",
+  MSRPOPL = "measure-population",
+  MSRPOPLEX = "measure-population-exclusion",
+  OBSERV = "measure-observation",
```
Comment (sb-prateekkeerthi) [inline src/api/QdmCalculationService.ts]: I see that all of them(except initialPopulation) are not following the camel case syntax is this expected ?

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

**Review by RohitKandimalla — DISMISSED** [review]

File: src/api/QdmCalculationService.ts (hunk 1 above)
Comment (nmorasb) [reply src/api/QdmCalculationService.ts]: so, I copied this over from the QDM execution written in EditTestCase for single test case execution. I should have checked that it was written properly, as it was NOT! So, I'll be updating these.
It works for IP, Denom and Numer, but anything hyphenated it fails.
I'm also adding an extra unit test with at least one hyphenated population.

Comment (nmorasb) [reply src/api/QdmCalculationService.ts]: this has been corrected and an extra unit test added

Comment (sb-prateekkeerthi) [reply src/api/QdmCalculationService.ts]: thanks for the clarification Nate

**Review by sb-prateekkeerthi — APPROVED** [review]

**Pull request merged**